### ISSUE TITLE
super expression must either be null or a function, not undifined

### ISSUE THREAD

**Issue by andyfen** [tomauty/react-native-chart#78]
i think ? this is because it needs react and component to be pulled from react package now

import React, { Component } from 'react';

**Comment by cablegunmaster** [comment]
oh this is a easy one, because of  the version of dev and gradle are not equal. 
in project: "cd android and type: ".sudo /gradlew clean" gonna reload gradle again (takes some time) if android server cannot be found try setting it manual.
`
ANDROID_HOME=/Where your SDK is / sudo ./gradlew clean
`
but its not the same issue? Actually it is, because its undefined and not knowing where to get the 'component' from because gradle Android can't find the component JS. I updated to the 0.27.2 native-react version. and ran the above command and got empty charts.

More info on this bug see:
http://stackoverflow.com/questions/19794200/gradle-android-and-the-android-home-sdk-location

I Tested Barchart, it seems very promising!
Gonna try to make some jest tests towarts this module to make it more robuust, and to understand the code more. Tested Barchart with react 15.1.0 and react-native 0.27.2  , no errors found for the barchart.

Keep up the good work! :+1:
